server/app.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+import { join } from 'path';
```

Comment:
Try to sort modules: 
* Core Modules
* Third Party Modules 
* Local Modules (your own modules)